parkings/api/monitoring/serializers.py

```diff
@@ -0,0 +1,27 @@
+import rest_framework_gis.serializers as gis_serializers
+from rest_framework import serializers
+
+from ...models import Parking
+
+
+class ParkingSerializer(gis_serializers.GeoFeatureModelSerializer):
+    operator_name = serializers.CharField(
+        source='operator.name', read_only=True)
+
+    class Meta:
+        model = Parking
+        fields = [
+            'id',
+            'registration_number',
+            'location',
+            # 'parking_area',
```

Comment:
Is this intentionally left here? And if it is, I don't really see the point.

Reply:
Yeah, there's no much use having it there as a comment. I will remove it.